README.md

```diff
@@ -37,7 +39,15 @@ When in doubt, use `snyk-to-html --help` or `snyk-to-html -h`.
 
 ## Generate the HTML report
 
-Snyk JSON to HTML Mapper mapper works with the different Snyk Products.  Change the directory to your package's root folder, then use  one of the  ways below to generate the HTML report, using the appropriate product's command
+Snyk JSON to HTML Mapper works with all of Snyk Products:
```

Comment:
I wouldn't say "all" products, because e.g. Snyk Cloud, Snyk Advisor are not supported.

Reply:
Also, for Snyk Container, application vulnerabilities are not supported, as they are reported in a different JSON element.